domains/encrypt0r.json

```diff
@@ -0,0 +1,10 @@
+{
+  "description": "main domain",
+  "owner": {
+    "username": "encrypt0r",
+    "email": "encrypt0r-x@protonmail.com"
+  },
+  "record": {
+    "A": "103.16.117.216"
```

Comment:
A record should be an array, also i wouldn't consider just this as complete
![image](https://github.com/user-attachments/assets/94d1682d-6913-4477-9887-7e727fcf9bd0)